data/answers.js

```diff
@@ -90,7 +90,9 @@
                     "helpfulness": 19,
                     "photos": []
                 }
-              }
             }
-  ]
-};
\ No newline at end of file
+        }
+    ]
+};
+
+export default answers;
```

Comment:
actually this export exports something called answers, but there's no answers variable defined in this file.  Perhaps at the top set the object to...
```suggestion
const answers = { ...
```